rfcs/min-relations.md

```diff
@@ -0,0 +1,270 @@
+# Feature Name: `min-relations`
+
+// FIXME
+// Make sure to talk about these:
+// - Reverse relations.
+// - Or support for add_target_filter.
+
+## Summary
+
+Relations connect entities to each other, storing data in these edges. This RFC covers the minimal version of this feature, allowing for basic entity grouping and simple game logic extensions.
```

Comment:
```suggestion
Relations allow creating logical connection between entity to multiple other entities. They also allow storing data associated with those individual connections, known as edges. Queries areexpanded to allow efficient filtering of entities that relate to specified group of entities.

This RFC covers the minimal version of this feature, allowing for basic dynamic entity grouping and simple game logic extensions.
```

Reply:
Relations allow users to create logical connection between entities. Each source entity can have multiple kinds of relations, each with their own data, and multiple relations of the same kind pointing to a different target entity.
Queries are expanded to allow efficient filtering of entities by the relations they have or the targets of these relations.